README.md

```diff
@@ -300,6 +305,9 @@ This command should be run once for every commercetool application.
 
 This command should be run once (or each time after `npm run api-extension-delete`).
 
+### Sync
+Data migration allows us to handle more advanced features of Voucherify, so it is important to keep data updated (however it is still optional if you use only basic functionalities). Migration is done via `npm run migrate` commands. When you are launching the integration app the first time you should fetch all data (`products`, `orders`, `customers`). After that to keep Voucherify updated it will be convenient to sync data once in a while. To not process all of the data each time you can pass additional arguments (e.g. `days`, `date`) to shorten the sync period and as a result decrease time of these operations. You can do it manually, but we highly recommend automating this process with some tool like `Cron`. The period between syncs should depend on traffic on your platform or the time when new vouchers, campaigns, etc. are created to make them as close to the newest data as possible.
```

Comment:
What metadata are we migrating? Do we migrate all of them? Do we need to configure it somehow in Voucherify?